btr-web/btr-main-app/components/individual-person/add-new/index.vue

```diff
@@ -14,34 +14,57 @@
       :show-section-has-errors="sectionErrors?.individualsFullName?.length > 0"
       :section-title="$t('sectionHeadings.individualsFullName')"
     >
-      <UForm
-        ref="profileFormBase"
-        :schema="formSchema"
-        :state="significantIndividual.profile"
-        class="w-full"
-        @change="addBtrPayFees"
-      >
-        <div class="flex-col w-full">
+      <div class="flex-col w-full">
+        <UForm
+          ref="profileFormName"
+          :schema="formSchema"
+          :state="significantIndividual.profile"
+          class="w-full"
+          @change="addBtrPayFees"
+        >
           <BcrosInputsNameField
             id="individual-person-full-name"
             v-model="significantIndividual.profile.fullName"
             name="fullName"
             :label="$t('labels.fullName')"
+            :placeholder="$t('placeholders.fullName')"
             :variant="fullNameInvalid ? 'error' : 'bcGov'"
             data-cy="testFullName"
           />
-          <div class="pt-3">
-            <BcrosInputsNameField
-              id="individual-person-preferred-name"
-              v-model="significantIndividual.profile.preferredName"
-              name="preferredName"
-              :label="$t('labels.preferredName')"
-              :variant="preferredNameInvalid ? 'error' : 'bcGov'"
-              data-cy="testPreferredName"
+        </UForm>
+        <UForm
```

Comment:
Was there no way around without adding additional form ?
We are trying to keep number of forms down to minimum, and eventually have only one if possible